js/script.js

```diff
@@ -0,0 +1,201 @@
+// ------------------------
+// LOGIC
+// ------------------------
+
+const Board = (() => {
+  let cells = new Array(9).fill(" ");
+
+  const fillCell = (cellNumber, cellSymbol) => {
+    cells[cellNumber] = cellSymbol;
+  }
+
+  const isCellTaken = (cellNumber) => {
+    return cells[cellNumber] != " ";
+  }
+
+  // const resetBoard = () => {
+  //   cells = new Array(9).fill(" ");
+  // }
+
+  return {cells, fillCell, isCellTaken}
+
+})();
+
+const Player = function(name, symbol) {
+  return {name, symbol}
+}
+
+const Game = (() => {
+  let moves = 0;
+  let playerX = Player("", "X");
+  let playerO = Player("", "O");
+  let currentPlayer = playerX;
+
+  // swap current player
+  const playerSwap = function() {
```

Comment:
NO! If  we go writing functions like this then, let the be arrow functions:
 `const playerSwap = () => { ... }`